ani-cli

```diff
@@ -63,12 +63,18 @@ help_info() {
         play dubbed version
       -U, --update
         Update the script
+      -Se --season
+        Select season (Needed for the -j --jellyfin option)
```

Comment:
Bad idea. `-Se` will get parsed by getopts as `-S` (select-nth) and `-e` (episode)